deploy/zfs-operator.yaml

```diff
@@ -22,6 +22,40 @@ spec:
     shortNames:
     - zfsvol
     - zv
+  validation:
+    openAPIV3Schema:
+      properties:
+        spec:
+          properties:
+            compression:
+              pattern: "^(on|off|lzjb|gzip|gzip-[1-9]|zle|lz4)$"
```

Comment:
@pawanpraka1 is there a default value to these fields?

Reply:
No @akhilerm, the way zfs works is if nothing is given while provisioning the volume it inherits the properties from pool. So here we don't need to have any default values.